### ISSUE TITLE
Would it be posssible to include a how-to on this in the readme? 

### ISSUE THREAD

**Issue by borpglass** [mitchellh/vagrant#5223]
I'm getting the same error described in https://github.com/mitchellh/vagrant/issues/1778 -- VERR_DISK_FULL -- but with an added twist on my stock ubuntu box. Running df -h shows this:

Filesystem Size Used Avail Use% Mounted on
/dev/sda5 77G 27G 47G 36% /
none 4.0K 0 4.0K 0% /sys/fs/cgroup
udev 973M 12K 973M 1% /dev
tmpfs 197M 1.1M 196M 1% /run
none 5.0M 0 5.0M 0% /run/lock
none 983M 84K 982M 1% /run/shm
none 100M 56K 100M 1% /run/user
/dev/sda1 69G 28G 42G 41% /media/data-desk-ubuntu/Default

(VMs appear to be stored in /home/data-desk-ubuntu/VirtualBox VMs )

Any ideas on where I could move that VMs folder to make Vagrant happy? It appears to already be in a place with 47G free (and only about 25G is required for the machine image I'm trying to run.)

I'd love to use ftdysa's helpful suggestion from that previous thread -- I just don't know where to relocate things, and I figure there's probably a pretty standard way to find it that I'm just not trying. Any tips would be met with shouts of praise. Any addition of those tips to the documentation -- maybe in an FAQ -- will be met with even more shouts of praise, as my ubuntu installation's rather fresh status makes me assume I'm not the only person having this type of problem.

**Comment by kikitux** [comment]
for new VMs you can use 

where `/new/path` is the folder you want to use

`mkdir -p </new/path>`

`VBoxManage setproperty machinefolder </new/path>`

or the gui

**Comment by borpglass** [comment]
Thanks, kitiux! That looks like helpful advice. 

FWIW, I ended up going with the advice here about changing default location: 

http://www.thisprogrammingthing.com/2013/changing-the-directory-vagrant-stores-the-vms-in/ 

Not sure whether this does the same thing. If not, I'll definitely fall back to your suggestion. I have yet to test it out, so I'll keep folks updated either way.

**Comment by borpglass** [comment]
Well, I tried both the VBoxManage setproperty machinefolder </new/path> and the link above (which, I suspect, both do the same thing), and things looked promising at first: VBoxManage indeed tried to work in the folder I specified (which, in turn, had been created in / -- the place with 47G free). But then I got this:

[default] Importing base box 'mybox'...
Progress: 60%There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["import", "/home/data-desk-ubuntu/.vagrant.d/boxes/mybox/virtualbox/box.ovf", "--vsys", "0", "--vmname", "mybox_0.52_1380817574_1422467399723_90386", "--vsys", "0", "--unit", "9", "--disk", "/big-folder-for-mybox/mybox_0.52_1380817574_1422467399723_90386/box-disk1.vmdk"]

Stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
Interpreting /home/data-desk-ubuntu/.vagrant.d/boxes/mybox/virtualbox/box.ovf...
VBoxManage: warning: The virtual system "mybox_0.52_1380817574" claims support for 8192 MB RAM size, but VirtualBox has support for min 4 & max 3584 MB RAM size only..
OK.
0%...10%...20%...30%...40%...50%...60%...
Progress state: VBOX_E_FILE_ERROR
VBoxManage: error: Appliance import failed
VBoxManage: error: Could not create the clone medium '/big-folder-for-mybox/mybox_0.52_1380817574_1422467399723_90386/box-disk1.vmdk'.
VBoxManage: error: VMDK: cannot write allocated data block in '/big-folder-for-mybox/mybox_0.52_1380817574_1422467399723_90386/box-disk1.vmdk' (VERR_DISK_FULL)
VBoxManage: error: Details: code VBOX_E_FILE_ERROR (0x80bb0004), component Appliance, interface IAppliance
VBoxManage: error: Context: "int handleImportAppliance(HandlerArg*)" at line 779 of file VBoxManageAppliance.cpp

... and running df -h still shows this:

Filesystem      Size  Used Avail Use% Mounted on
/dev/sda5        77G   27G   47G  37% /
none            4.0K     0  4.0K   0% /sys/fs/cgroup
udev            973M  4.0K  973M   1% /dev
tmpfs           197M  1.1M  196M   1% /run
none            5.0M     0  5.0M   0% /run/lock
none            983M   88K  982M   1% /run/shm
none            100M   48K  100M   1% /run/user
/dev/sda1        69G   28G   42G  41% /media/data-desk-ubuntu/Default

... so I'm back at square one. If nobody else has suggestions for diagnostics within the next few days, I'll probably wipe the host machine entirely and try dedicating the whole thing to Ubuntu, then seeing if that frees up enough space to solve this. Regardless of what happens, I'll keep this issue updated in case others end up running into the same problem and can benefit from my experience.

**Comment by borpglass** [comment]
Just to close the loop on this (sorta): We figured that dual-booting with Windows was causing some problems. So we rebooted from our Ubuntu 14.04LTS installation CD, re-ran the installer and chose this time to "erase everything and reinstall," which wiped out all the old Windows junk in the process. Other than turning on LVM for our new installation, we otherwise just used the default values suggested by the installation process. 

Once that installation was complete, we re-ran sudo apt-get install vagrant (and then the same for virtualbox when I realized it wasn't included by default as a dependency.) 

We then downloaded the vagrant box I wanted to run and got it up with no problems. So this specific situation has been remedied, even if I never found an actual "answer" to the extent that I would have liked.

(Oh, we also added a bunch of memory to the host machine -- bumping it from 1GB to 8GB of RAM -- but I don't think that necessarily had anything to do with the newfound success in booting my VM, as the perceived drive-space limitation appeared to be what was halting our progress before. I'm just including the RAM information to be thorough about every detail that we changed, in case it helps somebody else who's on the edge of sanity about this. Memory is cheap these days, so it certainly didn't hurt us to cover all the options.)
